tests/kernels/test_int8_quant.py

```diff
@@ -12,6 +12,18 @@
 SCALE = [0.1, 0.5, 0.8, 1.2, 2.1]
 
 
+def allclose_int(input, other, atol: int = 0, rtol: float = 1e-5):
+    INT_DTYPES = [
+        torch.int8, torch.int16, torch.int32, torch.int64, torch.uint8,
+        torch.uint16, torch.uint32, torch.uint64
+    ]
+    assert input.dtype in INT_DTYPES and other.dtype in INT_DTYPES
+    diff = torch.abs(input.to(torch.int64) - other.to(torch.int64))
+    return torch.all(
+        diff <= atol +
+        torch.ceil(rtol * torch.abs(other).to(torch.float32)).to(torch.int64))
```

Comment:
maybe comment on how the allclose works ?

Reply:
I think I only used this for testing, so I removed it